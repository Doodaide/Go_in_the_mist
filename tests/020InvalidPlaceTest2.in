who
place A12
who
place A12
placeD2
place d2
place  d2
place  D2
place_d2
PLACE d2
place DDDDDDD
place DDDDDDDDDDDDDDDDDDDDD
place D
place d
place D100
place d100
place D10 
place D10
who
term