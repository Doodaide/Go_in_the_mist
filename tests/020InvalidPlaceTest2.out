B
W
Occupied coordinate
Invalid!
Invalid coordinate
Invalid!
Invalid!
Invalid!
Invalid!
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid!
B
